### ISSUE TITLE
Webhook "customer.subscription.created" returning a 500 error

### ISSUE THREAD

**Issue by Murked** [laravel/cashier-stripe#1222]
- Cashier Version: 13.4
- Laravel Version: 8.22.1
- PHP Version: 8.0.0
- Database Driver & Version: MySQL 15.1

### Description:
So, i have setup laravel cashier & have been generating a checkout link/session & redirecting the user to stripe to make the payment. Everything is working as i would like it too, however when stripe send the webhooks back over to me (currently using stripe CLI to monitor this) im receiving two 500 http errors which give the following errors:

[Stripe CLI Log Image](https://gyazo.com/dda51cd70c58fe51486634f71b762fff)
[Stripe Dashboard Response Image](https://gyazo.com/80d9df3828c04a42c12b85c7dd14cd18)

Event: customer.subscription.created
Error: Call to a member function contains() on array in file C:\Users\Joe\Desktop\Projects\X\vendor\laravel\cashier\src\Http\Controllers\WebhookController.php on line 70

Event: customer.subscription.updated
Error: Call to a member function filter() on array in file C:\Users\Joe\Desktop\Projects\X\vendor\laravel\cashier\src\Http\Controllers\WebhookController.php on line 126

I am not entirely sure why this is triggering these errors, but since i didnt write any of this code, im not too sure what i should do to resolve the problem.

Do i need to update any of my versions?
Do i need to make any updates to any of my models?
Did i miss any obvious steps as to why this is causing me issues?

Should i just modify the source to use 'in_array' as a tempory fix?

Any information or help would be greatly appreciated as i seem to be stuck.

Thanks!

### Steps To Reproduce:
- Install Cashier
- Generate checkout link & redirect user. Make fake payment with fake card.
- Stripe webhook requests to default 'stripe/webook' URL logic to alert of payment creating error.

**Comment by driesvints** [comment]
> Error: Call to a member function contains() on array in file 

> Error: Call to a member function filter() on array in file 

These errors shouldn't be possible. Did you implement the Billable trait on your billable model? Can you share a repo that reproduces this?

**Comment by Murked** [comment]
Hey, yes i did implement the billable model. I actually found the issue later on after some debugging & it was a problem of my own. I had a multidimensional array in my user model called `$subscriptions` which turned the stripe plan ID into real product names & it seems that it was reading from this instead of the cashier stuff. I just moved this array into a config file instead & all was working as intended. 

Sorry for forgetting to close this issue, just thought i'd explain my problem in case anyone else experiences it. 

Thanks!
